Automattic/jetpack
# Social: Add the instagram social icon

With an icon available for instagram the connections should be displayed correctly in the editor sidebar.

## Proposed changes:
This updates the components to add an Instagram icon, and also the connection components so the `instagram-business` connections will be shown along with the others.

### Other information:


## Jetpack product discussion
1203820938137329-as-1204448525205753/f

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* With a test Jetpack site, sandbox the API and apply D111103-code
* Create a test Instagram connection
* Load the editor and check the connection is shown in the sidebar correctly

<img width="252" alt="image" src="https://github.com/Automattic/jetpack/assets/96462/53a6feb4-ca67-49d7-8796-8e37cf8ac24f">
